EstateReporting.IntegrationTests/Common/Setup.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace EstateReporting.IntegrationTests.Common
+{
+    using System;
+    using Ductus.FluentDocker.Services;
+    using Ductus.FluentDocker.Services.Extensions;
+    using global::Shared.Logger;
+    using NLog;
+    using Shouldly;
+    using TechTalk.SpecFlow;
+
+    [Binding]
+    public class Setup
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6339123)